### PR TITLE
Feature/responsive

### PR DESCRIPTION
**Pull request by MerlinEgalite**

### CODE REVIEW

File: src/views/level0/Level0-1.vue [hunk 1]
```diff
@@ -13,11 +13,12 @@
         class="x-input"
         @valid="forms.vset.input.name"
         v-model="forms.input.name"
-        placeholder="Enter your name"
+        placeholder="Enter your Telegram or Discord handle"
         required
       />
+      <button commons @click="goNext" class="mt-2" :disabled="!forms.vget.input.form">next</button>
+      <h2 class="teko-subtitle mt-1p5" style="cursor: pointer;" @click="$router.push('/intro')">watch intro again</h2>
```
Comment (mendesfabio) [inline src/views/level0/Level0-1.vue:20]: I talked to Marko and he said the `watch intro again` link should have the same style as the Exit and Help buttons.

**Review by mendesfabio — CHANGES_REQUESTED** [review]
Great work, @MerlinEgalite. After fixing the link, we can merge it.

File: src/views/level0/Level0-1.vue (hunk 1 above)
Comment (MerlinEgalite) [reply src/views/level0/Level0-1.vue:20]: oh okay!

Comment (MerlinEgalite) [reply src/views/level0/Level0-1.vue:20]: Screenshot 2021-01-24 at 15 45 18
then is it ok like that?

Comment (mendesfabio) [reply src/views/level0/Level0-1.vue]: yeah, I think so.

**Review by mendesfabio — APPROVED** [review]

**Pull request merged**